CHANGELOG.md

```diff
@@ -12,7 +12,25 @@ and this project adheres to:
 
 ## Unreleased
 
-## [0.0.1] - 2022-09-03
+### Added
+
+* Added Github Actions for (cargo check, cargo fmt, cargo clippy, cargo test) [John15321](https://github.com/John15321/)
+* Added CHANGELOG.md
+* Added `ids` module
+
+### Changed
+
+* Added LICENSE with MIT License by [John15321](https://github.com/John15321/)
+* Added CI Badges to Readme by [John15321](https://github.com/John15321/)
```

Comment:
The change itself should follow this format:
```
* Added something [#PRNUMBER](github.link/to/the/pr.com)
```

Reply:
Fixed the issue